dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/event/DefaultEventChangeLogService.java

```diff
@@ -128,6 +134,23 @@ public void addDataValueChangeLog(
     hibernateEventChangeLogStore.addEventChangeLog(eventChangeLog);
   }
 
+  @Override
+  @Transactional
+  public void addPropertyChangeLog(Event newEvent, Event currentEvent, String userName) {
```

Comment:
```suggestion
  public void addPropertyChangeLog(Event newEvent, Event currentEvent, String username) {
```

`username` is the field name.